system/modules/core/dca/tl_page.php

```diff
@@ -43,9 +43,9 @@
 			'keys' => array
 			(
 				'id' => 'primary',
-				'pid' => 'index',
 				'alias' => 'index',
-				'type' => 'index'
+				'type' => 'index', // FIXME: remove?
```

Comment:
As discussed on Mumble on March, 26th, the single index on the `type` column can be removed.